Gems/ScriptCanvas/Code/Editor/Components/GraphUpgrade.cpp

```diff
@@ -79,7 +79,7 @@ namespace ScriptCanvasEditor
         {
             if (node->GetComponents().empty())
             {
-                AZ_TracePrintf("Script Canvas", "Removing node due to missing components: %s\nVerify that all gems that this script relies on are enabled\n", node->GetName().c_str());
+                AZ_TracePrintf(ScriptCanvas::k_VersionExplorerWindow.data(), "Removing node due to missing components: %s\nVerify that all gems that this script relies on are enabled\n", node->GetName().c_str());
```

Comment:
minor: Using a string_view here makes the code less readable for no actual gain, a simple const char* should be enough